printrun/utils.py

```diff
@@ -190,7 +192,7 @@ def configfile(filename):
 def decode_utf8(s):
     try:
         s = s.decode("utf-8")
-    except:
```

Comment:
I have to admit I'm not familiar with the test suite. Do you mean the files in the folder 'testtools' ? But I agree that it is rather odd that the .decode throws AttributeErrors?

> I cannot reproduce a scenario where one would get a ValueError, and even if I could, would it be safe to ignore the error in such case?

`UnicodeDecodeError` is a sub-error of `ValueError`s and it should only occur when a character cannot be resolved as a unicode character.
The .decode() also has its own error handler that we could utilise instead of `try`:
bytes.decode(encoding='utf-8', errors='strict')
```
strict | Default behavior which raises UnicodeDecodeError on failure.
ignore | Ignores the un-encodable Unicode and continue without further notice.
replace | Replaces all un-encodable Unicode characters with a question mark. �
```

Your proposed change looks like an improvement to me! As far as I see, decode_utf8() is only used to decode error message strings of the operating system `strerror`. I did not know that this might be necessary.

Reply:
> `UnicodeDecodeError` is a sub-error of `ValueError`s and it should only occur when a character cannot be resolved as a unicode character.

Good point. I missed that entirely.